### DISCUSSION TITLE
Turn off Filters

### DISCUSSION THREAD

**Discussion by gregmax17** [pixijs/pixijs#7547] [Q&A]
Is there a way to disable filters? I have a project with many Display Objects and some have filters applied, but instead of going through the entire project and removing the filters applied manually, is there a way to just tell PIXI to skip the filter render process? Or maybe, maybe, allow specific filters to be rendered?

**Comment by bigtimebuddy** [comment]
All filters have an `enabled` property that you can set to be `false`. This will effectively skip that filter from being rendered. Good option if you need to do per-filter exclusion.

Pixi doesn't have a global switch to turn off all filters. But you could do a quick hack, like this to override setting filters from being applied. 

```js
renderer.filter.push = () => null;
renderer.filter.pop = () => null;
```
